### PR TITLE
Add "setup-cocoapods" action

### PR DESCRIPTION
**Pull request by maxim-lobanov**
Added a link to the "Setup Cocoapods" action:
- [Repository](https://github.com/maxim-lobanov/setup-cocoapods)
- [Marketplace](https://github.com/marketplace/actions/setup-cocoapods)
This action allows you to setup any version of Cocoapods on macOS and Ubuntu images.
It supports specifying Cocoapods version or specifying path to the Podfile.lock to resolve version dynamically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Actions for Logtalk and Prolog](https://github.com/logtalk-actions)
 - [GitHub Actions for Deno](https://github.com/denolib/setup-deno)
 - [GitHub Actions for Unity](https://github.com/webbertakken/unity-actions)
+- [GitHub Actions for Cocoapods](https://github.com/maxim-lobanov/setup-cocoapods)
```
Comment (jessicalostinspace) [inline README.md]: Again, although there are two different configurations, the underlying action is that it sets up cocoapods. Would you mind moving this to utility as well? I will make sure to add a "Mobile" section in the next week and move the appropriate actions there.

Comment (maxim-lobanov) [reply README.md]: Fixed, thank you for your feedback

**Comment by maxim-lobanov** [comment]
Hello @jessicalostinspace , please let me know we need to get this PR merged

**Comment by jessicalostinspace** [comment]
@maxim-lobanov please address merge conflicts and tag me when you're ready for review. Thank you :)

**Comment by maxim-lobanov** [comment]
@jessicalostinspace , Done

**Pull request merged**